packages/task/src/browser/taskNode.ts

```diff
@@ -0,0 +1,37 @@
+/********************************************************************************
+ * Copyright (C) 2019 Red Hat, Inc. and others.
```

Comment:
I believe this is the wrong company :)
In your case I think it should be updated to `SAP`.

Reply:
done, thanks